packages/example-app-base/src/components/Page/Page.types.ts

```diff
@@ -161,6 +161,11 @@ export interface IPageSectionProps<TPlatforms extends string = string>
 }
 
 /** Version of IPageSectionProps where `sectionName` is required. */
-export type IPageSectionPropsWithSectionName = Required<Pick<IPageSectionProps, 'sectionName'>> & Omit<IPageSectionProps, 'sectionName'>;
+// TODO: I'm not sure the best way to fix this, and the TS watch issue is making it harder to iterate and try fixes.
+//        Equating types with a slight loss in type safety for now.
+export type IPageSectionPropsWithSectionName = IPageSectionProps;
+// export type IPageSectionPropsWithSectionName<TPlatform extends string = string> =
+//   Required<Pick<IPageSectionProps<TPlatform>, 'sectionName'>> & Omit<IPageSectionProps<TPlatform>, 'sectionName'>;
+// export type IPageSectionPropsWithSectionName = Required<Pick<IPageSectionProps, 'sectionName'>> & Omit<IPageSectionProps, 'sectionName'>;
```

Comment:
@ecraig12345 FYI. The watch issue combined with my lack of knowledge with this code made it hard to iterate and find a good fix for this.